.github/workflows/AirspeedVelocity.yml

```
@@ -0,0 +1,78 @@
+name: Benchmark a Pull Request
+
+on:
+  pull_request_target:
+    branches:
+      - master
```

```suggestion
      - main
```
We use `main` instead of `master`. I think/hope this is also the reason why this Action didn't run for this PR.